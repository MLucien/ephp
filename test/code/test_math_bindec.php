<?php

class A {}

print bindec("101") . "\n";
print bindec("150") . "\n";
print bindec("aa") . "\n";
print bindec(array()) . "\n";
print bindec(100) . "\n";
print bindec(new A()) . "\n";
print "end\n";
