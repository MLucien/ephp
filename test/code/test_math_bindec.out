5
2
0

Notice: Array to string conversion in {{CWD}}/test/code/test_math_bindec.php on line 8
0
4

Catchable fatal error: Object of class A could not be converted to string in {{CWD}}/test/code/test_math_bindec.php on line 10
